docs/source/api/apollo-server.mdx

```diff
@@ -5,8 +5,6 @@ api_reference: true
 
 This API reference documents the `ApolloServer` class.
```

Comment:
I wonder if we can come up with a little more to say here. (Also "This API reference documents" seems a little weird.) Like just saying that this is the main thing you use to build your server, that you're expected to start it and connect it to a web framework (or use startStandaloneServer), etc. Could also be a reasonable place to remind folks that this comes from `@apollo/server` (and that any ApolloServer you've found in a package starting with `apollo-server` is AS3 or older).

Reply:
This is such a great point! I elaborated on this a bit and added a plug for the "Getting Started" guide:

>This article documents the `ApolloServer` class from the `@apollo/server` package. You can use the `ApolloServer` class to create an instance of Apollo Server that you can then pass to a web framework integration function (e.g., `startStandaloneServer` or `expressMiddleware`). 
>
> > Check out our [Getting Started](../getting-started) guide for an example of setting up Apollo Server!